### PR TITLE
Add docs for ocsp staple support

### PR DESCRIPTION
**Pull request by kfaseela**
API docs PR: https://github.com/istio/api/pull/2803

### CODE REVIEW

**Comment by kfaseela** [comment]
OCSP staple has been merged way back in 1.17 time, but the docs were not updated. See https://github.com/istio/istio/pull/43209/files However the CRL changes which we merged yesterday was only for 1.19

**Comment by ericvn** [comment]
If we merged doc changes that are only for 1.19, then yes we will want to revert. We can do that after we get past the branch cutting since I want to minimize PRs until that gets completed.

**Comment by kfaseela** [comment]
> If we merged doc changes that are only for 1.19, then yes we will want to revert. We can do that after we get past the branch cutting since I want to minimize PRs until that gets completed.

https://github.com/istio/istio.io/pull/13309

**Comment by ericvn** [comment]
Main branch is now set for 1.19 docs.

**Comment by kfaseela** [comment]
> Main branch is now set for 1.19 docs.

great! will wait for your review then.
cc @frankbu

File: content/en/docs/tasks/traffic-management/ingress/secure-ingress/index.md [hunk 1]
```diff
@@ -522,7 +522,10 @@ You can extend your gateway's definition to support [mutual TLS](https://en.wiki
       --from-file=ca.crt=example_certs1/example.com.crt
     {{< /text >}}
 
+    {{< tip >}}
     {{< boilerplate crl-tip >}}
+    The credential may also include [OCSP Staple](https://datatracker.ietf.org/doc/html/rfc6961) using the key `tls.ocsp-staple` which can be specified by an additional argument: `--from-file=tls.ocsp-staple=/some/path/to/your-ocsp-staple.pem`.
```
Comment (frankbu) [inline content/en/docs/tasks/traffic-management/ingress/secure-ingress/index.md:527]:
```suggestion

    The credential may also include an [OCSP Staple](https://datatracker.ietf.org/doc/html/rfc6961) using the key `tls.ocsp-staple` which can be specified by an additional argument: `--from-file=tls.ocsp-staple=/some/path/to/your-ocsp-staple.pem`.
```

**Review by frankbu — APPROVED** [review]

**Pull request merged**